modules/unsupported/cql-json/src/main/java/org/geotools/filter/text/cqljson/CQLJson.java

```diff
@@ -0,0 +1,76 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2016, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+
+package org.geotools.filter.text.cqljson;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.geotools.filter.text.commons.CompilerUtil;
+import org.geotools.filter.text.cql2.CQLException;
+import org.geotools.filter.text.cqljson.model.Predicates;
+import org.opengis.filter.Filter;
+import org.opengis.filter.FilterFactory;
+
```

Comment:
We try for public javadocs, this is an unsupported module so you can wait until this is ready to be consider for the library.